third_party/d3flamegraph/update.sh

```diff
@@ -21,36 +21,10 @@ D3FLAMEGRAPH_CSS="d3-flamegraph.css"
 
 cd $(dirname $0)
 
-generate_d3_flame_graph_go() {
+install_and_copy() {
```

Comment:
d3 based flame graph is going away in pprof - see #777.

So I would prefer to avoid any changes to it now.

Reply:
okay, so I will revert changes in `third_party/d3flamegraph` but will keep in `third_party/svgpan`. Can you confirm that?

Reply:
Yes, this sounds good.

Reply:
Done